bus/client.go

```diff
@@ -146,9 +149,51 @@ func (c *Client) SendSiacoins(ctx context.Context, scos []types.SiacoinOutput) (
 	return c.BroadcastTransaction(ctx, append(parents, txn))
 }
 
+// WalletTransactionsOption is an option for the WalletTransactions method.
+type WalletTransactionsOption func(url.Values)
```

Comment:
I think these options are better placed in the `api` package. Same as the `UploadOption` which'll be moved by one of our open PRs I think. Instead of formatting the time manually perhaps we should wrap them in `api.ParamTime` instead to be more consistent.

Reply:
I disagree. This type is not a request nor response type that we marshal or need to use on both sides of the API.
It's only meant to be used with a single method and as such it would be awkward to not be able to use that method without importing the param from a separate package.

Reply:
The idea of the `api` package is to avoid cyclic imports. If we keep the options in the packages, and want to use it in the autopilot for instance, the autopilot has to import the `bus` package... won't that get us into trouble?